vllm/utils.py

```diff
@@ -2176,7 +2176,33 @@ def _check_multiproc_method():
         os.environ["VLLM_WORKER_MULTIPROC_METHOD"] = "spawn"
 
 
-def get_mp_context():
+def get_mp_context(
```

Comment:
I think you can just set `VLLM_WORKER_MULTIPROC_METHOD` env var without changing this function.

Reply:
Then you need users to explicitly set the env variable right? We cannot set the env var before determining the executor class

Reply:
> We cannot set the env var before determining the executor class

any potential harm you see from the proposal? i.e. change `VLLM_WORKER_MULTIPROC_METHOD` to spawn if we are in a ray actor, no matter what is the executor class.

Reply:
My point is This function is doing the same thing. It calls `_check_multiproc_method()` that sets `VLLM_WORKER_MULTIPROC_METHOD` in certain cases. In other words, this function is exactly the place we set `VLLM_WORKER_MULTIPROC_METHOD`.